src/Features/LanguageServer/Protocol/ILspSolutionProvider.cs

```diff
@@ -15,7 +15,7 @@ internal interface ILspSolutionProvider
         /// </summary>
         /// <param name="documentUri">the document's file path URI.</param>
         /// <returns>the documents in the correct workspace and solution context</returns>
-        ImmutableArray<Document> GetDocuments(Uri documentUri);
+        ImmutableArray<Document> GetDocuments(Uri? documentUri);
```

Comment:
Between all of the extension methods that all call each other I'm having trouble tracking through why this changed, but its surprising that this is nullable. What happens when its null? Does it just return all documents?

The LspSolutionProviders job is to map documents to their solutions, so based on no document information its just going to fall back to assuming you want the VisualStudioWorkspace.